src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -559,7 +559,7 @@ private function formatControllerLink($controller, string $anchorText, ?callable
             } elseif (!\is_string($controller)) {
                 return $anchorText;
             } elseif (str_contains($controller, '::')) {
-                $r = new \ReflectionMethod($controller);
+                $r = \PHP_VERSION_ID < 80300 ? new \ReflectionMethod($controller) : \ReflectionMethod::createFromMethodName($controller);
```

Comment:
Actually the method is available since 8.3. I'd say let's use it with the lowest version possible?

Reply:
I agree. The earlier we can drop conditional code, the better.

Reply:
we don't need any condition if we use explode, isn't it?
aren't there more occurences of this pattern in the codebase? It'd be nice to have only one PR to fix the all.

Reply:
> we don't need any condition if we use explode, isn't it?

That is correct, but why wouldn't we want to leverage the native method if there is one? I like the current way because it hints at how the code will look like once we drop PHP 8.2 support (8.0 probably?).

Reply:
On my side I have a preference for not planning for more future PRs, preserving git blame, and saving a few possible conflicts :)

Reply:
I predict that someone will make that change on 8.0 anyway. :-D

But yeah, explode or condition, both works for me.

Reply:
I agree with the future-proof opinion of your solution. I updated the PR. Also, I could not find other places to update, `ReflectionMethod` seems always called with two arguments 👍